R/iSEE.R

```diff
@@ -331,15 +331,19 @@ iSEE <- function(
             currgraph_used <- delete.vertices(pObjects$brush_links,not_used)
             currgraph_used <- set_vertex_attr(currgraph_used,"plottype",
                                               value = gsub("Plot[0-9]","",V(currgraph_used)$name))
+            
+            currgraph_used <- .find_links_to_table(rObjects, pObjects, currgraph_used)
```

Comment:
It's good that you put `.find_links_to_table` in a separate file; could you move the entire graph-making function into `codetrack.R` as well? This should give us some breathing space with respect to the number of lines in `iSEE.R`.

Reply:
I aimed at a working version first, since me and `igraph` are beginner in terms of friendship level.
Got a version that works stably with more tables, proceeding to clean up

Reply:
Did the modification to `split_encoded` allow you to make more tables?

Reply:
done this one too 😉

Reply:
> Did the modification to split_encoded allow you to make more tables?
 
Yep!